docs/examples/grpc/app.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: fortune-teller-app
+  labels:
+    k8s-app: fortune-teller-app
+  namespace: default
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: fortune-teller-app
+    spec:
+      containers:
+      - name: fortune-teller-app
+        image: container.stack.build/pcj/fortune-teller:latest
```

Comment:
Should we host this in the repo together with echoserver and other tools @aledbf?

Reply:
That makes sense

Reply:
I was planning on putting this in quay.io, but they still don't support docker registry v2.2 so I decided against it (ref https://github.com/bazelbuild/rules_docker/issues/102 gentle ping @philips).

```sh
E0502 07:46:27.210516   20444 docker_session_.py:335] Error during upload of: quay.io/pcj/fortune-teller:latest
F0502 07:46:27.210813   20444 fast_pusher_.py:162] Error publishing quay.io/pcj/fortune-teller:latest: response: {'status': '415', 'content-length': '131', 'server': 'nginx/1.13.6', 'connection': 'keep-alive', 'date': 'Wed, 02 May 2018 13:46:27 GMT', 'content-type': 'application/json'}
manifest invalid: {u'message': u'manifest schema version not supported'}
```

Reply:
@aledbf Is hosting this image as @antoineco suggests part of this PR?

Reply:
Yes please

Reply:
> Can I just mkdir images/grpc-fortune-teller

Yes. After merging the PR I will publish the image in the same place than the ingress image

Reply:
What will the anticipated image name be? (for the `app.yaml` image name).  Something under quay.io, gcr.io/google_containers, or elsewhere?

Reply:
`quay.io/kubernetes-ingress-controller/grpc-fortune-teller:0.1`

Reply:
Done